src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -346,6 +346,9 @@ private void DynamoView_Loaded(object sender, EventArgs e)
             sidebarGrid.Children.Add(search);
             dynamoViewModel.SearchViewModel.Visible = true;
 
+            var incanvasSeachViewModel = new SearchViewModel(dynamoViewModel, dynamoViewModel.Model.SearchModel);
```

Comment:
Incanvas search should have its own SeachViewModel. If it has the same ViewModel as LibraryView, both of them will show the same result.